test/test_svd.cc

```diff
@@ -57,6 +58,30 @@ void test_svd_work( Params& params, bool run )
     params.ortho_V();
     params.error.name( "S - Sref" );
     params.error2.name( "Backward" );
+    if (timer_level >= 2) {
+        params.time2();
+        params.time3();
+        params.time4();
+        params.time5();
+        params.time6();
+        params.time7();
+        params.time8();
+        params.time9();
+        params.time10();
+        params.time11();
+        params.time12();
+        params.time2.name( "geqrf (s)" );
+        params.time3.name( "gelqf (s)" );
+        params.time4.name( "ge2tb (s)" );
+        params.time5.name( "tb2db (s)" );
+        params.time6.name( "bdsvd (s)" );
+        params.time7.name( "unmbr_tb2st_U (s)" );
+        params.time8.name( "unmbr_ge2tb_U (s)" );
+        params.time9.name( "unmqr (s) " );
+        params.time10.name( "unmbr_tb2st_V (s)" );
```

Comment:
Typo: should be `tb2bd` instead of `tb2st`. Bidiagonal (bd) happens in the SVD, symmetric tridiagonal (st) happens in eig. In a few places:
```
pangolin ~/Documents/slate> git grep unmbr_tb2st
test/test_svd.cc:        params.time7.name( "unmbr_tb2st_U (s)" );
test/test_svd.cc:        params.time10.name( "unmbr_tb2st_V (s)" );
test/test_svd.cc:            params.time10() = slate::timers[ "svd::unmbr_tb2st_V" ];
```